<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-26T09:59:21 IST" name="Default suite" finished-at="2024-03-26T09:59:27 IST" duration-ms="6696">
    <groups>
      <group name="Login">
        <method signature="Main.login()[pri:2, instance:opensooq_selenium.Main@53f0a4cb]" name="login" class="opensooq_selenium.Main"/>
      </group> <!-- Login -->
      <group name="Home">
        <method signature="Main.Home()[pri:0, instance:opensooq_selenium.Main@53f0a4cb]" name="Home" class="opensooq_selenium.Main"/>
      </group> <!-- Home -->
    </groups>
    <test started-at="2024-03-26T09:59:21 IST" name="Default test" finished-at="2024-03-26T09:59:27 IST" duration-ms="6696">
      <class name="opensooq_selenium.Main">
        <test-method is-config="true" signature="setup()[pri:0, instance:opensooq_selenium.Main@53f0a4cb]" started-at="2024-03-26T09:59:21 IST" name="setup" finished-at="2024-03-26T09:59:21 IST" duration-ms="194" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Home()[pri:0, instance:opensooq_selenium.Main@53f0a4cb]" started-at="2024-03-26T09:59:21 IST" name="Home" finished-at="2024-03-26T09:59:23 IST" duration-ms="2560" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Home -->
        <test-method signature="HomePage_En()[pri:0, instance:opensooq_selenium.Main@53f0a4cb]" started-at="2024-03-26T09:59:23 IST" name="HomePage_En" finished-at="2024-03-26T09:59:26 IST" duration-ms="2113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HomePage_En -->
        <test-method signature="login()[pri:2, instance:opensooq_selenium.Main@53f0a4cb]" started-at="2024-03-26T09:59:26 IST" name="login" finished-at="2024-03-26T09:59:27 IST" duration-ms="1525" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at opensooq_selenium.Main.login(Main.java:80)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="closepage()[pri:0, instance:opensooq_selenium.Main@53f0a4cb]" started-at="2024-03-26T09:59:27 IST" name="closepage" finished-at="2024-03-26T09:59:27 IST" duration-ms="225" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closepage -->
      </class> <!-- opensooq_selenium.Main -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
